tsconfig.json

```diff
@@ -20,7 +16,5 @@
     "noEmit": true,
     "jsx": "react-jsx"
   },
-  "include": [
-    "src"
-  ]
+  "include": ["src", "**/*.ts", "**/*.tsx", "**/*.js", "**/*.jsx"]
```

Comment:
Is this intentional change? I suppose it worked before, why is this necessary now?